modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,479 @@
+:_module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-manage-objects-cli.adoc
+
+[id="rosa-create-objects_{context}"]
+= Create objects
+
+[role="_abstract"]
+This section describes the `create` commands for clusters and resources.
+
+[id="rosa-create-admin_{context}"]
+== create admin
+
+Creates a cluster administrator with an automatically generated password that can log in to a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create admin --cluster=<cluster_name>
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to add to the identity provider (IDP).
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+.Example
+Create a cluster administrator that can log into a cluster named `mycluster`.
```

Comment:
s/log into/log in to
**log in to** _v_
Write as shown - three words.

Reply:
Done.